setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import setup, find_packages
+
+setup(name='project1',
+      version='1.0',
+      packages=find_packages()
+      )
```

Comment:
Please lint the code using Flake8 (I think it all looks fine, except an empty line at the end) and include the appropriate comments that I've added to the sample code in #2.

I also prefer when you the close bracket is on the same tab as the initial command. Also, if you use new lines for each parameter, please put them all on new lines and do a single tab inwards. This makes the code a little more readable as you don't have parameters and method names jumbled on one line. Like this
```python
setup(
    name='project1',
    version='1.0',
    packages=find_packages()
)
```

Reply:
running `flake8` found 

```
setup.py:7:1: W293 blank line contains whitespace
setup.py:7:7: W292 no newline at end of file
```

@bradley-erickson , I ran flake8 and lint the setup.py.

Reply:
Awesome, I'm assuming these are taken care of. Thanks